lineage/figures/figure9.py

```diff
@@ -20,59 +20,58 @@ def makeFigure():
     lapatinib = [Lapatinib_Control + Gemcitabine_Control, Lapt25uM, Lapt50uM, Lap250uM]
     gemcitabine = [Lapatinib_Control + Gemcitabine_Control, Gem5uM, Gem10uM, Gem30uM]
 
-    def find_AIC(data, desired_num_states):
+    def find_AIC(data, desired_num_states, num_cells):
         # Copy out data to full set
         dataFull = []
         for _ in desired_num_states:
             dataFull.append(data)
 
         # Run fitting
         output = run_Analyze_over(dataFull, desired_num_states, atonce=True)
-        AICs = np.array([oo[0][0].get_AIC(oo[2], atonce=True)[0] for oo in output])
+        AICs = np.array([oo[0][0].get_AIC(oo[2], num_cells, atonce=True)[0] for oo in output])
 
         return AICs - np.min(AICs, axis=0)
 
-    lapAIC = find_AIC(lapatinib, desired_num_states)
-    gemAIC = find_AIC(gemcitabine, desired_num_states)
+    lapAIC = find_AIC(lapatinib, desired_num_states, num_cells=5290)
+    gemAIC = find_AIC(gemcitabine, desired_num_states, num_cells=4537)
 
-    # what is the best number of states
-    lpt_st = desired_num_states[np.argmin(lapAIC)]
-    gmc_st = desired_num_states[np.argmin(gemAIC)]
+    # # Lapatinib
+    # lapt_tHMMobj_list, lapt_states_list, _ = Analyze_list(lapatinib, 3, fpi=True)
 
-    # run analysis for the found number if states
-    lapt_tHMMobj_list, lapt_states_list, _ = Analyze_list(lapatinib, lpt_st, fpi=True)
-    gemc_tHMMobj_list, gemc_states_list, _ = Analyze_list(gemcitabine, gmc_st, fpi=True)
+    # # assign the predicted states to each cell
+    # for idx, lapt_tHMMobj in enumerate(lapt_tHMMobj_list):
```

Comment:
Is this commented code still needed?

Reply:
Not now because we have the pickled files, but if for any reason we need to run the analysis again, we will. Should I remove it?